src/utils/configuration.ts

```diff
@@ -154,11 +154,14 @@ async function tryToReadClassInformationFromDecorators(
     projectConfiguration: YamlProjectConfiguration,
 ) {
     const cwd = projectConfiguration.workspace?.backend || process.cwd();
+
     const allJsFilesPaths = (await getAllFilesFromPath(cwd)).filter((file: FileDetails) => {
+        const folderPath = path.join(cwd, file.path);
         return (
             (file.extension === ".js" || file.extension === ".ts") &&
             !file.path.includes("node_modules") &&
-            !file.path.includes(".git")
+            !file.path.includes(".git") &&
+            !fs.lstatSync(folderPath).isDirectory()
```

Comment:
I think you could have just given the `file.path` here and remove the need of adding a new variable.

```suggestion
            !fs.lstatSync(file.path).isDirectory()
```